### PR TITLE
TACO-104 Add Lotame pixel

### PR DESCRIPTION
**Pull request by jakubbanas**
Lotame pixel integration

### CODE REVIEW

File: src/platforms/ucp-desktop/modes/ucp-desktop-ads.mode.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {
 	jwPlayerInhibitor,
 	LiveConnect,
 	LiveRampPixel,
+	Lotame,
```
Comment (domjanik) [inline src/platforms/ucp-desktop/modes/ucp-desktop-ads.mode.ts:20]: What about other platforms? UCP-mobile, N&R etc?

File: src/ad-services/lotame/index.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			return;
+		}
+		utils.logger(this.logGroup, 'pixel enabled');
+		await utils.scriptLoader.loadScript(this.PIXEL_URL);
+		communicationService.dispatch(eventsRepository.LOTAME_LOADED);
+
+		const lotameTagInput = {
+			data: {},
+			config: {
+				clientId: Number(this.CLIENT_ID),
```
Comment (domjanik) [inline src/ad-services/lotame/index.ts:27]: CLIENT_ID is number anyway, therefore I think it's pointless to do that conversion.

File: src/ad-services/lotame/index.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	async call(): Promise<void> {
+		if (!this.isEnabled('icLotame')) {
+			utils.logger(this.logGroup, 'pixel disabled');
+			return;
+		}
+		utils.logger(this.logGroup, 'pixel enabled');
+		await utils.scriptLoader.loadScript(this.PIXEL_URL);
+		communicationService.dispatch(eventsRepository.LOTAME_LOADED);
+
+		const lotameTagInput = {
```
Comment (domjanik) [inline src/ad-services/lotame/index.ts:24]: Shouldn't we create that config object BEFORE loading a pixel ? 🤔

Comment (jakubbanas) [reply src/ad-services/lotame/index.ts:20]: By documentation, it doesn't seem to matter, but it looks like a better idea.
It still doesn't work, but looks better :D

File: src/ad-services/lotame/index.ts (hunk 2 above)
Comment (jakubbanas) [reply src/ad-services/lotame/index.ts]: Good point

File: src/platforms/ucp-desktop/modes/ucp-desktop-ads.mode.ts (hunk 1 above)
Comment (jakubbanas) [reply src/platforms/ucp-desktop/modes/ucp-desktop-ads.mode.ts:20]: Right, forgot about it

**Pull request merged**